ihmc-perception/src/main/java/us/ihmc/perception/RawImage.java

```diff
@@ -105,6 +106,30 @@ public RawImage(RawImage other)
       this.orientation = other.orientation;
    }
 
+   public static RawImage fromMessage(ImageMessage imageMessage)
```

Comment:
I think this'll be useful for the YOLO detections, specifically getting the object mask image.